src/Database/Redis/PubSub.hs

```diff
@@ -83,6 +101,14 @@ sendCmd cmd       = do
     lift $ Core.send (redisCmd cmd : changes cmd)
     modifyPending (updatePending cmd)
 
+totalPendingChanges :: PubSub -> Int
+totalPendingChanges (PubSub{..}) =
+  updatePending subs $ updatePending unsubs $ updatePending psubs $ updatePending punsubs 0
```

Comment:
I'd say that sum of pending counts would look more intuitive here but that's just a minor nitpicking...